### DISCUSSION TITLE
Графические настройки

### DISCUSSION THREAD

**Discussion by Georgiy-Timoshin** [OGSR/OGSR-Engine#448] [General]
1. С версии 2.1.402 были убраны настройки: Дальность видимости, Качество текстур, Фильтрация текстур и др. Разве этих настроек нет в современных играх? Зато есть тонкие настройки глубины резкости, высоты травы, раздельные настройки угла обзора. 

2. Часть настоек, которые не влияют на производительность, можно было бы вынести из подвкладки "расширенных" в новую, например "дополнительные" или во вкладку "Игра". 

3. Окно настроек и окно со списком сохранений можно увеличить.

Хотелось бы сделать более удобным интерфейс.

**Comment by I-AmDead** [comment]
1. Это те настройки что на вкус и цвет
2.Например какие
3.Сделай, каждый свой интерфейс делает

**Comment by xrSimpAI** [comment]
> Дальность видимости, Качество текстур, Фильтрация текстур 

Дальность видимости вообще менять нельзя, т.к. это регулируется погодой. Уменьшая её, погода перестаёт выглядеть так как задумывалось автором.
А уменьшая качество текстур и фильтрацию, текстуры просто превращаются в мыло. Какой смысл играть в такое мыло? В современных играх нет настроек, уменьшив которые можно превратить игру в настолько ужасный вид.
